manual/cypher/cypher-docs/src/docs/graphgists/query-tuning/advanced-query-tuning-example.asciidoc

```diff
@@ -9,27 +9,27 @@ As with the basic query tuning example <<cypherdoc-basic-query-tuning-example>>
 //setup
 [source,cypher]
 ----
-LOAD CSV WITH HEADERS FROM "movies.csv" AS line
-MERGE (m:Movie {title:line.title})
+LOAD CSV WITH HEADERS FROM 'movies.csv' AS line
+MERGE (m:Movie {title: line.title})
 ON CREATE SET m.released = toInt(line.released), m.tagline = line.tagline
 ----
 
 //setup
 [source,cypher]
 ----
 LOAD CSV WITH HEADERS FROM 'actors.csv' AS line
-MATCH (m:Movie {title:line.title})
-MERGE (p:Person {name:line.name})
+MATCH (m:Movie {title: line.title})
+MERGE (p:Person {name: line.name})
 ON CREATE SET p.born = toInt(line.born)
-MERGE (p)-[:ACTED_IN {roles:split(line.roles,";")}]->(m)
+MERGE (p)-[:ACTED_IN {roles: split(line.roles,';')}]->(m)
```

Comment:
The style guide is missing rules for function arguments, such as whether they should be space-separated. I'll open an issue at the repo.